Makefile

```diff
@@ -45,7 +44,8 @@ deploy-artefacts:
 .PHONY: format-cfn
 format-cfn:
 	$(eval VERSION := $(shell $(MAKE) -s version))
-	sed -i '' -e '3s/.*/Description: Amazon S3 Find and Forget \(uksb-1q2j8beb0\) \(version:$(VERSION)\)/' templates/template.yaml
+	sed -i.bak -e '3s/.*/Description: Amazon S3 Find and Forget \(uksb-1q2j8beb0\) \(version:$(VERSION)\)/' templates/template.yaml
```

Comment:
Nitpick: you should use mktemp when a temporary filename is needed - otherwise this could overwrite/delete an existing file.

Reply:
Thanks for the advice, i have done as suggested